orttraining/orttraining/python/training/ortmodule/_custom_gradient_registry.py

```diff
@@ -276,3 +277,45 @@ def upsample_nearest3d_gradient():
 @register_gradient("org.pytorch.aten", "ATen", "upsample_bicubic2d", "vec")
 def upsample_bicubic2d_gradient():
     return _upsample_gradient("upsample_bicubic2d_backward", 2)
+
+
+ATEN_SDPA_FALLBACK = os.getenv("ORTMODULE_ATEN_SDPA_FALLBACK", None)
```

Comment:
this will be tricky when some of them attention is masked, and the others are not masked, we cannot differentiate them.,

Reply:
as per Vincent's suggestion, I am removing support for masked attention and focusing this solution only on unmasked which is what stable diffusion utilizes.

Reply:
if so, we'd better add an assert attn_mask is None in  
```def scaled_dot_product_attention(g, query, key, value, attn_mask=None, dropout_p=0.0, is_causal=False, scale=None):```

if assertion fail, then we are aware of it clearly. And shall we add some notes we only support attn_mask is None?